tcms/testcases/migrations/0011_trim_bugsystem_fields.py

```diff
@@ -1,6 +1,37 @@
 # Generated by Django 2.2.4 on 2019-08-16 12:17
 
-from django.db import migrations
+from django.db import migrations, models
+
+
+def forwards_store_data(apps, schema_editor):
+    bug_system_model = apps.get_model('testcases', 'BugSystem')
+    with open('/tmp/bugsystems-migration-dump-0011-\
+trim-bugsystem-fields',  # nosec:B108:hardcoded_tmp_directory
```

Comment:
Filename doesn't follow convetion like the rest. Use `/tmp/kiwitcms-<app_name>-migration-<number>-<model_name_being_altered>`